### ISSUE TITLE
model

### ISSUE THREAD

**Issue by cxsunny** [smu-ivpl/FastInterCU-VVC#3]
Can the model pt file in the file be used directly? Why can't I run it?

**Comment by unacanal** [comment]
Yes. What kind of error occured? Have you checked the version of TorchScript?

**Comment by cxsunny** [comment]
I have solved this problem. However, there are still some unclear points in the paper. Why should we add the predicted results of the model after checking the merge mode rdcost? What is the logic?

**Comment by unacanal** [comment]
The target of this work is to reduce complexity of INTER modes, so the residual image is acquired right after Merge mode as the best CU at that point.

**Comment by cxsunny** [comment]
How to get the prediction image of CTU? Can you share the code of making data set?

Thank you very much.
